core/lib/network-request.js

```diff
@@ -363,26 +382,29 @@ class NetworkRequest {
     if (timing.requestTime === 0 || timing.receiveHeadersEnd === -1) return;
     // Take startTime and responseReceivedTime from timing data for better accuracy.
     // Timing's requestTime is a baseline in seconds, rest of the numbers there are ticks in millis.
-    // TODO: This skips the "queuing time" before the netstack has taken over ... is this a mistake?
-    this.startTime = timing.requestTime;
-    const headersReceivedTime = timing.requestTime + timing.receiveHeadersEnd / 1000;
-    if (!this.responseReceivedTime || this.responseReceivedTime < 0) {
-      this.responseReceivedTime = headersReceivedTime;
+    this.networkRequestTime = timing.requestTime * 1000;
+    const headersReceivedTime = this.networkRequestTime + timing.receiveHeadersEnd;
```

Comment:
wait, is it actually when the last header byte is received?

Reply:
⚠️ ‼️ ⚠️ ‼️ ⚠️ ‼️ ⚠️ ‼️ 

there's no "start" equivalent on the timing object... It seems in practice it wouldn't make a difference, surely all sites flush out all the headers upfront 

does the lack of `receiveHeadersStart` imply that blink doesn't "stream" headers? if a preload header comes on the main document...then the server blocks before doing more headers then the body....would Chrome delay that preload?

Reply:
> wait, is it actually when the last header byte is received?

yes

I've poked at this with the netlog, but i've never had a testcase where the response headers get split over two packets. 
But.. if i remember right even the //net stack doesnt really call out headersStart as a thing.